docs/WASI-api.md

```diff
@@ -347,10 +347,11 @@ Inputs:
 
     The file descriptor to inspect.
 
-- <a href="#fd_filestat_get.buf" name="fd_filestat_get.buf"></a><code>[\_\_wasi\_filestat\_t](#filestat) \*<strong>buf</strong></code>
+Outputs:
 
-    The buffer where the file's attributes are
-    stored.
+- <a href="#fd_filestat_get.buf" name="fd_filestat_get.buf"></a><code>[\_\_wasi\_filestat\_t](#filestat) <strong>buf</strong></code>
+
+    The file's attributes.
```

Comment:
This change does make sense. Note that the witx file has already been changed along these lines in the WASI repo: https://github.com/WebAssembly/WASI/blob/master/phases/ephemeral/witx/wasi_ephemeral_preview.witx#L137 (though the return type is incorrect, so I've now submitted https://github.com/WebAssembly/WASI/pull/141 to fix it).